scripts/perftest/genTreeData.sh

```diff
@@ -36,32 +36,32 @@ echo "NOT DONE YET. WAITING FOR DML SCRIPT FROM FARAZ" ; exit 1
 
 
 #generate XS scenarios (80MB)
-${CMD} -f ../datagen/genRandData4LogisticRegression.dml $DASH-args 10000 1000 5 5 $BASE/w10k_1k_dense $BASE/X10k_1k_dense $BASE/y10k_1k_dense 1 0 $DENSE_SP $FORMAT
```

Comment:
@asurve `genRandData4LogisticRegression` has the argument `tranform_labels` which is missing in `genTreeData.sh`. Could you please help here?.